crates/polars-pipe/src/pipeline/mod.rs

```diff
@@ -28,6 +28,8 @@ pub(crate) fn determine_chunk_size(n_cols: usize, n_threads: usize) -> PolarsRes
         val.parse().map_err(
             |_| polars_err!(ComputeError: "could not parse 'POLARS_STREAMING_CHUNK_SIZE' env var"),
         )
+    } else if n_cols == 0 {
+        Ok(1)
     } else {
         let thread_factor = std::cmp::max(12 / n_threads, 1);
         Ok(std::cmp::max(50_000 / n_cols * thread_factor, 1000))
```

Comment:
Instead of special-casing `n_cols == 0` could you instead just change this line to

    Ok(std::cmp::max(50_000 / n_cols.max(1) * thread_factor, 1000))

?

Reply:
You are right, updated.